pkg/installer/Dockerfile

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) 2024 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+# Dockerfile to build installer img initrd
+FROM lfedge/eve-alpine:c114cf1d3ea51534f061f9aa949beb6ac5c12fb3 AS build
+SHELL ["/bin/ash", "-eo", "pipefail", "-c"]
+ENV BUILD_PKGS mkinitfs grep patch make coreutils musl-dev gcc g++ perl \
+    autoconf automake libtool file bsd-compat-headers libc-dev \
+    util-linux-dev flex linux-headers glib-dev libxrandr-dev zlib-dev \
+    libusb-dev kmod-dev curl eudev-dev libdrm-dev i2c-tools-dev hwinfo \
+    binutils-dev libaio-dev zlib-static libvncserver-dev python3 pixman-dev \
+    py3-setuptools bash perl cmake m4 pkgconf pkgconfig autoconf-archive tpm2-tss-dev \
+    cryptsetup-dev go lddtree
+ENV PKGS mtools dosfstools libarchive-tools sgdisk e2fsprogs util-linux \
+    squashfs-tools coreutils tar dmidecode smartmontools libaio libaio-dev \
+    perl glib zlib libusb curl xz pciutils usbutils hdparm util-linux \
+    wireless-tools libxrandr eudev-libs libusb sudo fio iperf3 sysstat \
+    lm-sensors acpi iw libdrm hwinfo dhclient dmidecode smartmontools \
+    pciutils libgcc pixman glib libvncserver musl-utils dhclient \
+    qemu-system-x86_64 tpm2-tss-dev tpm2-tss-esys tpm2-tss-fapi tpm2-tss-rc \
```

Comment:
Do we really need qemu-system-x86_64 in the installer?...

Reply:
I don't know. But I don't want to change too much from how current installer+verifier works or what it has. The goal in this one is to replace the process for getting a working installer image (and clean up all of the complexity on the way, which includes how we build and grub and ipxe etc)

If this was there before, and doesn't break anything, let's leave it. We always can do another PR that just touches `pkg/verifier` when done.

Reply:
Mmm... It wasn't obvious to me that this came from an old version. I thought it was a new file.

Reply:
Yeah, you are right, it wasn't clear. It consolidates everything that was in both the old installer and the old verification.